app/Resources/translations/validators.en.yml

```diff
@@ -23,4 +23,5 @@ title.field.article.files.not.be.blank: 'Title fields of article files can not b
 file.field.article.files.not.be.blank: 'File fields of article files can not be blank.'
 user.multiple_mail.invalid: 'This email is already registered'
 user.multiple_mail.error.not_email: 'You must enter a valid email'
-article.status_can_not_be_published_without_author: 'You must to add author before pull article status to published'
\ No newline at end of file
+article.status_can_not_be_published_without_author: 'You must to add author before pull article status to published'
+user.invalid.mail.domain: 'Mail have invalid domain address'
```

Comment:
_Invalid_ implies that user's email domain is not a proper domain. I think we should rather let the user know that their domain is banned.